src/ucs/datastruct/arbiter.c

```diff
@@ -91,6 +91,55 @@ void ucs_arbiter_group_purge(ucs_arbiter_t *arbiter, ucs_arbiter_group_t *group,
     group->tail = NULL;
 }
 
+void ucs_arbiter_group_purge_cond(ucs_arbiter_t *arbiter,
```

Comment:
need unit test in test_arbiter.cc for ucs_arbiter_group_purge_cond
also can we implement ucs_arbiter_group_purge() using ucs_arbiter_group_purge_cond()?

Reply:
We would need to add additional restriction to the current purge API, because purge description says: `Callback return value is ignored.`  So, I'd leave two functions (plus better code readability)

Reply:
i'd prefer to unite code to increase test coverage
can we migrate code which uses ucs_arbiter_group_purge to ucs_arbiter_group_purge_code?